src/bsp/src/stm32/src/HostUart.cpp

```diff
@@ -0,0 +1,137 @@
+#include "HostUart.h"
+#include "hal/uart_host.h"
+#include <FreeRTOSConfig.h>
+#include <cstdio>
+#include <cstring>
+#include <task.h>
+
+void hostUart_C_txCpltCallback(void* hostUartInstance) {
+    static_cast<HostUart*>(hostUartInstance)->txCpltCallback();
+}
+
+void hostUart_C_rxCpltCallback(void* hostUartInstance) {
+    static_cast<HostUart*>(hostUartInstance)->rxCpltCallback();
+}
```

Comment:
Maybe static since it's limited to the scope of this file

Reply:
Doesn't work if we make it a friend of the HostUart class. Leaving it as is.